Broken links on website

The website build has a few broken links that need fixing:

```
website-1  | - Broken link on source page path = /docs/cli/cloud/migrating/:
website-1  |    -> linking to /slack/
website-1  |    -> linking to ../../language/settings/backends/configuration.mdx/ (resolved as: /docs/cli/language/settings/backends/configuration.mdx/)
```

This needs to be backported to v1.7 and v1.6.

```
Run npm run lint

> website@0.0.0 lint
> eslint --cache "**/*.{js,jsx,ts,tsx}"

Warning: React version not specified in eslint-plugin-react settings. See https://github.com/jsx-eslint/eslint-plugin-react#configuration .

/home/runner/work/opentofu.org/opentofu.org/src/components/Accordion/Item.tsx
Warning:   98:6  warning  React Hook useEffect has missing dependencies: 'id', 'isHashEnabled', and 'location.hash'. Either include them or remove the dependency array  react-hooks/exhaustive-deps

/home/runner/work/opentofu.org/opentofu.org/src/theme/DocItem/Layout.tsx
Error:   5:8  error  'DocVersionBadge' is defined but never used  @typescript-eslint/no-unused-vars

/home/runner/work/opentofu.org/opentofu.org/versioned_docs/version-main/intro/install/buildkite.jsx
Error:   11:16  error  Component definition is missing display name  react/display-name
Error:   11:18  error  'children' is defined but never used          @typescript-eslint/no-unused-vars
Error:   11:28  error  'color' is defined but never used             @typescript-eslint/no-unused-vars

/home/runner/work/opentofu.org/opentofu.org/versioned_docs/version-v1.6/intro/install/buildkite.jsx
Error:   4:16  error  Component definition is missing display name  react/display-name
Error:   4:18  error  'children' is defined but never used          @typescript-eslint/no-unused-vars
Error:   4:28  error  'color' is defined but never used             @typescript-eslint/no-unused-vars

/home/runner/work/opentofu.org/opentofu.org/versioned_docs/version-v1.7/intro/install/buildkite.jsx
Error:   11:16  error  Component definition is missing display name  react/display-name
Error:   11:18  error  'children' is defined but never used          @typescript-eslint/no-unused-vars
Error:   11:28  error  'color' is defined but never used             @typescript-eslint/no-unused-vars

✖ 11 problems (10 errors, 1 warning)

Error: Process completed with exit code 1
```

> ```
> Run npm run lint
> 
> > website@0.0.0 lint
> > eslint --cache "**/*.{js,jsx,ts,tsx}"
> 
> Warning: React version not specified in eslint-plugin-react settings. See https://github.com/jsx-eslint/eslint-plugin-react#configuration .
> 
> /home/runner/work/opentofu.org/opentofu.org/src/components/Accordion/Item.tsx
> Warning:   98:6  warning  React Hook useEffect has missing dependencies: 'id', 'isHashEnabled', and 'location.hash'. Either include them or remove the dependency array  react-hooks/exhaustive-deps
> 
> /home/runner/work/opentofu.org/opentofu.org/src/theme/DocItem/Layout.tsx
> Error:   5:8  error  'DocVersionBadge' is defined but never used  @typescript-eslint/no-unused-vars
> 
> /home/runner/work/opentofu.org/opentofu.org/versioned_docs/version-main/intro/install/buildkite.jsx
> Error:   11:16  error  Component definition is missing display name  react/display-name
> Error:   11:18  error  'children' is defined but never used          @typescript-eslint/no-unused-vars
> Error:   11:28  error  'color' is defined but never used             @typescript-eslint/no-unused-vars
> 
> /home/runner/work/opentofu.org/opentofu.org/versioned_docs/version-v1.6/intro/install/buildkite.jsx
> Error:   4:16  error  Component definition is missing display name  react/display-name
> Error:   4:18  error  'children' is defined but never used          @typescript-eslint/no-unused-vars
> Error:   4:28  error  'color' is defined but never used             @typescript-eslint/no-unused-vars
> 
> /home/runner/work/opentofu.org/opentofu.org/versioned_docs/version-v1.7/intro/install/buildkite.jsx
> Error:   11:16  error  Component definition is missing display name  react/display-name
> Error:   11:18  error  'children' is defined but never used          @typescript-eslint/no-unused-vars
> Error:   11:28  error  'color' is defined but never used             @typescript-eslint/no-unused-vars
> 
> ✖ 11 problems (10 errors, 1 warning)
> 
> Error: Process completed with exit code 1
> ```

Hi, @cam72cam  I could be wrong, it doesn't look like this is happening because of the Slack link I changed?

--- a/website/docs/intro/install/buildkite.jsx
+++ b/website/docs/intro/install/buildkite.jsx
@@ -8,8 +8,21 @@
 import React from 'react';
 import BuildKiteSVG from './buildkite.svg'
 
-export default ({children, color}) => (
-    <p style={{textAlign: "center", padding: "1.5rem"}}><a href={"https://buildkite.com"} style={{display: "block", color: "#fff", textDecoration: "none"}}>
-        Thank you to <BuildKiteSVG style={{maxWidth: "50%", marginLeft: "auto", marginRight: "auto"}}/> for sponsoring the OpenTofu package hosting.
-    </a></p>
-);
+const BuildKite = () => {
+    return (
+      <p style={{ textAlign: "center", padding: "1.5rem" }}>
+        <a
+          href={"https://buildkite.com"}
+          style={{ display: "block", color: "#fff", textDecoration: "none" }}
+        >
+          Thank you to{" "}
+          <BuildKiteSVG
+            style={{ maxWidth: "50%", marginLeft: "auto", marginRight: "auto" }}
+          />{" "}
+          for sponsoring the OpenTofu package hosting.
+        </a>
+      </p>
+    );
+};
+
+export default BuildKite;

--- a/website/docs/intro/migration/terraform-1.5-or-lower.mdx
+++ b/website/docs/intro/migration/terraform-1.5-or-lower.mdx
@@ -121,7 +121,7 @@ If you encountered a bug, [please report it on GitHub](https://github.com/opento
 
 ## Troubleshooting
 
-If you encounter any issues during the migration to OpenTofu, you can join the [OpenTofu Slack](/slack/) or ask on
+If you encounter any issues during the migration to OpenTofu, you can join the <a href="/slack/">OpenTofu Slack</a> or ask on
 [GitHub Discussions](https://github.com/orgs/opentofu/discussions).
 
 ### Error: Failed to query available provider packages

--- a/website/docs/intro/migration/terraform-1.6.mdx
+++ b/website/docs/intro/migration/terraform-1.6.mdx
@@ -136,7 +136,7 @@ If you encountered a bug, [please report it on GitHub](https://github.com/opento
 
 ## Troubleshooting
 
-If you encounter any issues during the migration to OpenTofu, you can join the [OpenTofu Slack](/slack/) or ask on
+If you encounter any issues during the migration to OpenTofu, you can join the <a href="/slack/">OpenTofu Slack</a> or ask on
 [GitHub Discussions](https://github.com/orgs/opentofu/discussions).
 
 ### Error: Failed to query available provider packages

--- a/website/docs/intro/migration/terraform-1.7.mdx
+++ b/website/docs/intro/migration/terraform-1.7.mdx
@@ -148,7 +148,7 @@ If you encountered a bug, [please report it on GitHub](https://github.com/opento
 
 ## Troubleshooting
 
-If you encounter any issues during the migration to OpenTofu, you can join the [OpenTofu Slack](/slack/) or ask on
+If you encounter any issues during the migration to OpenTofu, you can join the <a href="/slack/">OpenTofu Slack</a> or ask on
 [GitHub Discussions](https://github.com/orgs/opentofu/discussions).
 
 ### Error: Failed to query available provider packages

--- a/website/docs/intro/migration/terraform-1.8.mdx
+++ b/website/docs/intro/migration/terraform-1.8.mdx
@@ -147,7 +147,7 @@ If you encountered a bug, [please report it on GitHub](https://github.com/opento
 
 ## Troubleshooting
 
-If you encounter any issues during the migration to OpenTofu, you can join the [OpenTofu Slack](/slack/) or ask on
+If you encounter any issues during the migration to OpenTofu, you can join the <a href="/slack/">OpenTofu Slack</a> or ask on
 [GitHub Discussions](https://github.com/orgs/opentofu/discussions).
 
 ### Error: Failed to query available provider packages

--- a/website/docs/intro/upgrading.mdx
+++ b/website/docs/intro/upgrading.mdx
@@ -100,7 +100,7 @@ If you encountered a bug, [please report it on GitHub](https://github.com/opento
 
 ## Troubleshooting
 
-If you encounter any issues during the migration to OpenTofu, you can join the [OpenTofu Slack](/slack/) or ask on
+If you encounter any issues during the migration to OpenTofu, you can join the <a href="/slack/">OpenTofu Slack</a> or ask on
 [GitHub Discussions](https://github.com/orgs/opentofu/discussions).
 
 ### Error: Failed to query available provider packages

